dali/imgcodec/util/exif.h

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_IMGCODEC_UTIL_EXIF_H_
+#define DALI_IMGCODEC_UTIL_EXIF_H_
+
+#include "dali/imgcodec/image_format.h"
+
+namespace dali {
+namespace imgcodec {
+
+/**
+ * @brief Sets orientation field in ImageInfo according to EXIF orientation code
+ */
+void SetOrientation(ImageInfo &info, uint16_t exif_orientation);
```

Comment:
I don't like the "surgeon" design pattern. Perhaps we should have a standalone orientation type and return it? Chances are that we might want to parse EXIF in a different scenario, where we don't even have an ImageInfo.

Reply:
Done